src/test/java/edu/kis/powp/jobs2d/window/command/CommandManagerWindow.java

```diff
@@ -22,11 +26,17 @@
 	private String selectedFilePath;
 
 	private ICommandManagerController controller;
+	private DriverCommandManager commandManager;
```

Comment:
We introduced the controller interface to remove coupling with app model (e.g. features and managers).  I hope we can avoid bringing back the dependency with DriverCommandManager. Please try to fix this.